docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -12,173 +12,40 @@ videoBanner: UFhT52d5bYg
 - AWS account with a Redshift cluster and permissions to create and attach IAM
   policies.
 
-## Step 1/5. Install Teleport
+## Step 1/6. Install Teleport
 
 (!docs/pages/includes/database-access/start-auth-proxy.mdx!)
 
-## Step 2/5. Create Teleport user
+## Step 2/6. Create Teleport user
 
 (!docs/pages/includes/database-access/create-user.mdx!)
 
-## Step 3/5. Configure IAM
+## Step 3/6. Create database agent configuration
 
-### Create an IAM policy for Teleport
+(!docs/pages/includes/database-access/token.mdx!)
+
+Create the Database Service configuration file:
+
+```code
```

Comment:
Since we refer to the flags after the snippet and the values must be changed for the user's environment, I think we should put each flag on a different line so the reader can see all of them.

Reply:
I updated the commands.